server/src/bin/main.rs

```diff
@@ -34,9 +35,29 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         })
         .transpose()?;
 
+    let remove_quote_callback = Arc::new(Mutex::new(|_quotes: Vec<Quote>| { /* Do nothing */ }));
+    // let remove_quote_callback: RemoveQuoteCallback =
+    //     Arc::new(Mutex::new(move |quotes: Vec<Quote>| {
+    //         for quote in quotes {
+    //             callback_msg_bus_tx
+    //                 .blocking_send(Msg::SciQuoteRemoved(*quote.id()))
+    //                 .unwrap_or_else(|_| {
+    //                     panic!(
+    //                         "Failed to send SCI quote {} removed message to
+    // message bus",                         quote.id()
+    //                     )
+    //                 });
+    //         }
+    //     }));
```

Comment:
let's remove this commented code if it's not needed?

Reply:
How did you test this?
It seems like if you are not posting to the message bus, removals will not make their way to GRPC LiveUpdate streamers.

Reply:
Sorry. Yes. This was my bad. I commented this out when I did the merge from main, and forgot to go back and fix it.